lib/internal/event_target.js

```diff
@@ -59,7 +63,12 @@ const kBubbles = Symbol('bubbles');
 const kComposed = Symbol('composed');
 const kPropagationStopped = Symbol('propagationStopped');
 
-const isTrusted = () => false;
+const isTrustedSet = new SafeWeakSet();
+const isTrusted = ObjectGetOwnPropertyDescriptor({
+  get isTrusted() {
+    return isTrustedSet.has(this);
+  }
+}, 'isTrusted').get;
```

Comment:
```suggestion
function isTrusted() {
  return isTrustedSet.has(this);
}
```

Reply:
That will make the `isTrusted` getter function have an own `prototype` property and the `[[Construct]]` internal method.

It will also cause it to have the wrong `name`.

Reply:
@ExE-Boss would you mind explaining why this doesn't work?

Reply:
Because the following will no longer throw a `TypeError`:
```js
new (Object.getOwnPropertyDescriptor(new Event('foo'), "isTrusted").get)();
```

Reply:
We should add a test for this across the EventTarget properties :]